lib/commands/misc/rmrole.js

```diff
@@ -0,0 +1,53 @@
+const { Command } = require('discord.js-commando');
+const { GENERAL_ID } = require('../../constants.js');
+const { getChannel } = require('./../../util.js');
+
+module.exports = class RemoveRoleCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'rmrole',
+			group: 'misc',
+			memberName: 'rmrole',
+			description: 'Remove a self-assigned role from yourself.',
+			details: '',
+			args: [
+				{
+					key: 'role',
+					prompt: 'Remove a role!',
+					type: 'string',
+					infinite: false,
+					wait: 1
+				}
+			]
+		});
+		this.queues = {};
+	};
+
+  async run(msg, args) {
+    const channel = this.client.channels.get(getChannel(REOL_CHAN_ID));
+    const role = msg.guild.roles.find(r => r.name == args.role);
```

Comment:
I feel like we should be checking if we have a value for `args.role` first, just in case the user has only issued the command with no arguments. (although this could be handled internally, I'm not sure)